config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -44,15 +44,16 @@ periodics:
     preset-e2e-scalability-node: "true"
   annotations:
     testgrid-dashboards: sig-scalability-experiments
-    testgrid-tab-name: max-emptydir-vol-per-pod
+    testgrid-tab-name: storage-single-node-gce
   spec:
     containers:
     - image: gcr.io/k8s-testimages/kubekins-e2e:v20190801-9ec6db7-master
       args:
       - --repo=k8s.io/kubernetes=master
       - --repo=k8s.io/perf-tests=master
+      - --repo=k8s.io/test-infra=master
       - --root=/go/src
-      - --timeout=60
+      - --timeout=80m
```

Comment:
does this need to be even longer since all tests now run in this one config?

Reply:
The original time was pretty conservative this one should be enough